birdhouse/config/geoserver/config/proxy/conf.extra-service.d/geoserver.conf.template

```diff
@@ -1,5 +1,5 @@
     location /geoserver/ {
-        proxy_pass http://${PAVICS_FQDN}:8087;
+        proxy_pass http://geoserver:8080/geoserver/;
```

Comment:
That bypasses the whole Magpie/Twitcher auth. It should be adjusted to use the protected proxy location while we are modifying this.

Reply:
Yes I didn't make any changes to which services were behind twitcher in this PR. As we discussed in #328 that feels like a change that should happen in a different PR.

Reply:
Note that the monitoring routes were an exception to that rule since putting those behind twitcher was the only way I could think to not expose the monitoring ports and still protect those routes.

Reply:
Yes. It's ok if done through a subsequent PR. I just want to make sure we don't forget about it.

Reply:
https://github.com/bird-house/birdhouse-deploy/issues/333